ui/src/tugboat/components/ModelCard.tsx

```diff
@@ -81,7 +81,7 @@ export const ModelCard = () => {
             </Descriptions.Item>
 
             <Descriptions.Item label="Citation" span={3}>
-                <Markdown>{models.selectedModel.card.citation || 'Unknown'}</Markdown>
+                {models.selectedModel.card.citation || 'Unknown'}
```

Comment:
I've been using the nullish coallesce
```suggestion
                {models.selectedModel.card.citation ?? 'Unknown'}
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

I like using nullish-coalesce as it's more explicit that it's not meant to be a boolean expression.